### ISSUE TITLE
Audio is muted after taking headset off and putting back on

### ISSUE THREAD

**Issue by chrisplatorres** [supermedium/moonrider#80]
On Oculus Quest: audio is muted after taking headset off and putting back on. (standby mode)

To fix it requires clicking to oculus home and then resuming oculus browser to hear any audio again.

**Comment by ngokevin** [comment]
Does it exit VR for you? If not, then maybe we just have to detect, and exit VR / pause. Might be the audio interaction gesture crap.

**Comment by chrisplatorres** [comment]
Yes, pressing "resume" on the Oculus Browser will no longer be in VR mode.

**Comment by JustinB4** [comment]
This happens to me occasionally as well.  An easy fix is to use the Oculus button, select quit, then go back into VR mode and the audio is fixed.
